lib/llm_composer/providers/bedrock.ex

```diff
@@ -0,0 +1,94 @@
+if Code.ensure_loaded?(ExAws) do
```

Comment:
I also added this condition to avoid ex_aws mandatory

CC @hectorperez

Reply:
me copié de la lib de tesla https://github.com/elixir-tesla/tesla/blob/c52eec88a478b899aa25894a02ec14045fd25aca/lib/tesla/adapter/finch.ex#L1